msteams-platform/m365-apps/prerequisites.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Set up your dev environment for extending Teams apps across Microsoft 365
+description: Here are the prerequisites for extending your Teams apps across Microsoft 365
+ms.date: 11/15/2021
+ms.topic: how-to
+---
+# Set up your dev environment for extending Teams apps across Microsoft 365 (preview)
+
+The development environment for extending Teams apps across Microsoft 365 is similar to what you use for Microsoft Teams development. This article discusses the configurations required for your test tenant to run preview builds of Microsoft Teams and Microsoft Office applications. These changes are needed because this feature is in preview.
+
+## Overview
+> [!div class="checklist"]
+> * Obtain an M365 Developer (Sandbox) Tenant and enable sideloading
+> * Enroll your M365 tenant in *Office 365 Targeted Releases*
+> * Configure your account to access preview versions of Outlook and Office apps
+> * Install Teams Toolkit vscode extension (optional)
+
+## Prepare a Developer Tenant for testing
+
+If you don't already have one, create a [Microsoft 365 developer subscription](/office/developer-program/microsoft-365-developer-program-get-started) sandbox tenant or obtain a test tenant through your organization.
+
+Once you have a tenant, you'll need to [enable sideloading](/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading) for your tenant by signing in to [Microsoft 365 admin center](https://admin.microsoft.com) and navigating to Show All > Teams > Teams apps > Setup policies > **Global**.  From there, toggle on **Upload custom apps** and **Save**.
```

Comment:
Global:

Please avoid usage of "once"

Reference: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/once

Reply:
Ah, nice catch! I learned something new :)